ang/civicase.ang.php

```diff
@@ -163,6 +158,13 @@ function set_case_actions(&$options, $caseCategoryPermissions) {
       'icon' => 'fa-trash',
     ];
   }
+  if (CRM_Core_Permission::check(CRM_Civicase_Service_ExportPermission::NAME)) {
```

Comment:
You can make the permission name a constant on the `CRM_Civicase_Hook_Permissions_ExportCasesAndReports` class. We don't need an extra service class.